privaterelay/locales/en-US/app.ftl

```diff
@@ -184,9 +185,9 @@ banner-download-firefox-cta = Get { -brand-name-firefox }
 banner-download-install-extension-headline = Looks like you haven’t installed the extension!
 banner-download-install-extension-copy = The { -brand-name-relay } extension for { -brand-name-firefox-browser } makes creating aliases even easier.
 banner-download-install-extension-cta = Get the extension for { -brand-name-firefox }
-banner-upgrade-headline = Upgrade { -brand-name-relay } for more features.
-banner-upgrade-copy = Upgrading gets you unlimited email addresses and your own email domain.
-banner-upgrade-cta = Upgrade { -brand-name-relay }
+banner-upgrade-headline = Buy { -brand-name-mozilla-privacy-pack } for more protection!
```

Comment:
Note that because this project is not yet actively being localized, these IDs do not need to be revised. If it were, these IDs would need to be updated too.